src/Connection/TermObjects.php

```diff
@@ -52,15 +52,15 @@ public static function register_connections() {
 		/** @var \WP_Taxonomy[] $allowed_taxonomies */
 		$allowed_taxonomies = \WPGraphQL::get_allowed_taxonomies( 'objects' );
 
-		/** @var \WP_Post_Type[] $allowed_post_types*/
-		$allowed_post_types = \WPGraphQL::get_allowed_post_types( 'objects' );
-
 		/**
 		 * Loop through the allowed_taxonomies to register appropriate connections
 		 */
 		foreach ( $allowed_taxonomies as $tax_object ) {
+			if ( ! $tax_object->graphql_register_root_connection ) {
+				continue;
+			}
 
-			$root_query_from_field_name = $tax_object->graphql_plural_name;
+			$root_query_from_field_name = lcfirst( $tax_object->graphql_plural_name );
```

Comment:
good catch. 

should this instead use `\WPGraphQL\Utils\Utils::format_field_name( $tax_object->graphql_plural_name )`?